src/app/cmdoptions.cpp

```diff
@@ -385,11 +385,11 @@ QStringList QBtCommandLineParameters::paramList() const
     if (firstLastPiecePriority)
         result.append(QLatin1String("@firstLastPiecePriority"));
 
-    if (skipDialog == TriStateBool::True) {
-        result.append(QLatin1String("@skipDialog=1"));
-    }
-    else if (skipDialog == TriStateBool::False) {
-        result.append(QLatin1String("@skipDialog=0"));
+    if (skipDialog) {
+        if (skipDialog.get())
```

Comment:
IMO you should decide on one: `operator*` or `value()`, I saw both.

Reply:
We can't use value() since it requires a newer version of boost. Probably better to use operator* then, since get() is deprecated.